app/resources/views/pages/home.blade.php

```diff
@@ -1,6 +1,20 @@
 @extends('layouts.default')
-@section('content')
+@section('head-content')
+	<script>
+			var user_longitude = {{ $base_user->getLongitude() }};
+			var user_latitude  = {{ $base_user->getLatitude() }};
+			var address_default = "{{ $address_default }}";
```

Comment:
I think address_default is unused because the placeholder from the input is used to retrieve it.  If that's correct, remove the address_default variable.

The user_longitude and user_latitude should be removed too since the Google Maps api can be used to retrieve the longitude and latitude off the current address.